hooks/pre_validations/12-check_capsule_tar.rb

```diff
@@ -4,8 +4,12 @@ def error(message)
   kafo.class.exit 101
 end
 
-if param('capsule', 'certs_tar') && (certs_tar = param('capsule', 'certs_tar').value)
-  unless File.file?(certs_tar)
-    error "The certs tar file generated by the server is not present at #{certs_tar}, exiting."
+certs_tar = param('foreman_proxy_certs', 'certs_tar') ||
```

Comment:
The module ```foreman_proxy_certs``` doesn't exist. Did you mean ```foreman_proxy_content```? This also looks like a bonus fix of previously dead code due to the module rename so 👍

Reply:
In capsule certs generated, there is mapping defined `https://github.com/Katello/katello-installer/blob/master/bin/foreman-proxy-certs-generate#L63` therefor it got here as well. Perhaps @ehelms knows why the mapping is there?

Reply:
Looking at it, I can't figure a real reason other than just typoing it during conversion away from capsule.